app/main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str = "") -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills = SoftwareEngineer.skills + ["JavaScript", "HTML", "CSS"]
+
+    def __init__(self, name: str = "") -> None:
+        super().__init__(name)
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    skills = SoftwareEngineer.skills + ["Python", "SQL", "Django"]
```

Comment:
u don't need to hardcode skills like that
just extend skills in the constructor method